.github/actions/sync-snippet-action/main.js

```diff
@@ -106,7 +105,7 @@ const run = async () => {
     ...repoInfo,
     title: PR_TITLE,
     head: branchName,
-    base: GITHUB_REF, // 'refs/heads/master' if action is running on master branch
+    base: 'master', // hard-coding this value, rather than use GITHUB_REF env var as a work-around to a bug in GitHub
```

Comment:
Um, is `GITHUB_REF` a _constant_ that is always `refs/heads/master`? The original comment makes it sound like it may not be. In which case, I think we need more logic here, so we only use hard-coded `master` when the `GITHUB_REF == "refs/heads/master"`.

Reply:
We'll only ever want to run the Action on `master` since this represents the latest version of the snippet that is deployed to `npm`. Technically, `GITHUB_REF` could reference a different branch, if the Action workflow is explicitly configured to run on a branch other than master (details here: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#onschedule)

Reply:
Ok.